playbooks/roles/foreman_provisioning/tasks/main.yml

```diff
@@ -0,0 +1,226 @@
+---
+# Get the smart proxy ID of the local katello:
+- name: 'get smart proxy id'
+  shell: >
+    {{ foreman_provisioning_hammer }} --output json proxy info --name {{ foreman_provisioning_proxy_name }}
+  register: foreman_provisioning_smart_proxy_json
+
+- name: 'set smart proxy id'
+  set_fact:
+    foreman_provisioning_smart_proxy: "{{ foreman_provisioning_smart_proxy_json.stdout|from_json }}"
+
+- name: 'install libvirt gem'
+  yum: name=rubygem-ruby-libvirt state=present
+
+- name: 'install foreman-libvirt'
+  yum: name=foreman-libvirt state=present
+
+- name: 'enable provisioning with foreman-installer'
+  shell: >
+    foreman-installer
+    --foreman-proxy-dns true
+    --foreman-proxy-dns-provider libvirt
+    --foreman-proxy-dns-interface virbr1
+    --foreman-proxy-dns-zone example.com
+    --foreman-proxy-dns-forwarders 8.8.8.8
+    --foreman-proxy-dns-reverse {{ foreman_provisioning_proxy_dns_reverse }}
+    --foreman-proxy-dhcp true
+    --foreman-proxy-dhcp-provider libvirt
+    --foreman-proxy-libvirt-network provision
+    --foreman-proxy-dhcp-interface virbr1
+    --foreman-proxy-dhcp-range "{{ foreman_provisioning_dhcp_start }} {{ foreman_provisioning_dhcp_end }}"
+    --foreman-proxy-dhcp-gateway {{ foreman_provisioning_ip_address }}
+    --foreman-proxy-dhcp-nameservers {{ foreman_provisioning_ip_address }}
+    --foreman-proxy-tftp-servername {{ foreman_provisioning_ip_address }}
+    --foreman-proxy-tftp true
+    --foreman-proxy-tftp-managed false
+    {{ foreman_provisioning_installer_options | join(" ") }}
+
+- name: 'restart foreman-proxy'
```

Comment:
Honestly, I don't know. I just moved the code from here https://github.com/theforeman/forklift/pull/399/files#diff-1574be18ceccad8d4354975e4cf6be59L29 to another location. I suppose the author of the task had a reason to add it. If we want to do a complete audit of all tasks in the roles, I'd treat it as a separate issue.